### DISCUSSION TITLE
any plans to addresslatest CVEs thanks

### DISCUSSION THREAD

**Discussion by SteveS7777** [eclipse-rdf4j/rdf4j#4260] [General]
any plans to address the 2 latest jackson-databind CVEs? Apcache Commons-text CVE?  thanks

**Comment by hmottestad** [comment]
Which versions are affected? What is the minimum version we need to upgrade to?

**Comment by SteveS7777** [comment]
for Jackson-databind it is 2.13.4.2, for apache commons text 1.10.0.

thank you




________________________________
From: Håvard M. Ottestad ***@***.***>
Sent: Monday, October 31, 2022 4:27 PM
To: eclipse/rdf4j ***@***.***>
Cc: Steve Serra ***@***.***>; Author ***@***.***>
Subject: [External] : Re: [eclipse/rdf4j] any plans to addresslatest CVEs thanks (Discussion #4260)


Which versions are affected? What is the minimum version we need to upgrade to?

**Comment by barthanssens** [comment]
Thanks for the heads-up. The Apache Commons Text is at 1.10.0 in the main tree, so should be in the next patch release

**Reply by barthanssens** [reply]
Also upgrading to jackson 2.13.4 and jackson-databind 2.13.4.2
